claudio-plugin/skills/gws-calendar-reader/tests/test_arg_validation.py

```
@@ -0,0 +1,169 @@
+import pathlib
+import subprocess
+
+import pytest
+
+_SCRIPTS = pathlib.Path(__file__).parent.parent / "scripts"
+TEAM_AGENDA = _SCRIPTS / "team_agenda.sh"
+CHECK_FREEBUSY = _SCRIPTS / "check_freebusy.sh"
+
+
+def run(script, *args):
+    return subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)